semver_tests/src/test_cases/non_exhaustive.rs

```diff
@@ -79,3 +79,16 @@ pub enum MyEnum {
     #[non_exhaustive]
     StructVariant { a: u64 },
 }
+
+#[cfg(not(feature = "enum_marked_non_exhaustive"))]
+pub enum SimpleEnum {
+    Foo,
+    Bar,
+}
+
+#[cfg(feature = "enum_marked_non_exhaustive")]
+#[non_exhaustive]
+pub enum SimpleEnum {
+    Foo,
+    Bar,
+}
```

Comment:
nitpick: leave a newline at the end of the file, you can probably configure your editor to do this automatically
```suggestion
}

```